proto/tendermint/abci/types.proto

```diff
@@ -127,11 +127,11 @@ message RequestApplySnapshotChunk {
 }
 
 message RequestPrepareProposal {
-  // block_data is an array of transactions that will be included in a block,
-  // sent to the app for possible modifications.
-  // applications can not exceed the size of the data passed to it.
+  // BlockData is a slice of candidate transactions that may be included in a
```

Comment:
[nit] 
```suggestion
  // block_data is a slice of candidate transactions that may be included in a
```

Reply:
I would rather use `BlockData` because IMO more people are likely to read the generated Go comments instead of the raw protobuf comments. 

![Screenshot 2024-06-17 at 10 09 07 AM](https://github.com/celestiaorg/celestia-core/assets/3699047/f7a1c213-a97d-4558-8e26-b38f08a9b619)